app/controllers/organizations_controller.rb

```diff
@@ -3,6 +3,8 @@ class OrganizationsController < ApplicationController
   rescue_from Errno::ENAMETOOLONG, with: :log_image_data_to_datadog
 
   def create
+    rate_limit!
```

Comment:
Any reason to not put this in a `before_action`? Seems like we wouldn't even want to continue with this request if the rate limiter decides that we've exceeded the limit?

Reply:
It is only being used in this method so I didn't put it in a before action. I think a refactor is due for this were this `rate_limit!` method could possibly be pulled out of individual controllers and put in application controller or the Rate limiter itself.

Reply:
This still feels pretty before-action-y to me, and I'd still vote for it to be pulled out (we have plenty of before_actions for only one method). But if you plan to refactor it into the application controller, I think that works too, and if that means that we'll overhaul all of the rate limiting stuff into the application controller, I think that is definitely preferable.